doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb

```diff
@@ -424,14 +424,14 @@
     "instantaneous_temperature = np.zeros(sampling_iterations)\n",
     "etotal = np.zeros(sampling_iterations)\n",
     "\n",
-    "for i in range(1, sampling_iterations + 1):\n",
+    "for i in range(0, sampling_iterations):\n",
```

Comment:
starting from ```0``` is the default and it is good style to omit the default value here.

Reply:
Fixed in a571818